### DISCUSSION TITLE
Mails (manually to groups of customers) are not sent and neither an error log about it is created.

### DISCUSSION THREAD

**Discussion by holgerm** [pretix/pretix#4507] [Q&A]
### Problem and impact

For a given evnt with 20 customers I wanted to send an email to all non-cancelled orders. Previously that worked for other events. 

Now we had to change our password on the mail server for the sending account. Since we forgot to update the password in pretix, of course the mails have not been sent, but error messages have been protocoled (visible via menu "User Settings --> Konto-Änderungen {do not know in english}). So far all ok.

Now we updated the password in pretix, got a green message that connection to the mail server is working and tried to send the mails again to our customers. This time, no error messages were protocoled. But the mails have not been sent. 

In order to be sure, I as admin created a fake order to get the mails myself too. Then I tried to sent a mail to new customers (just me). But again neither I got the mail, nor teh incident of failure got noted among the "Konto-Änderungen".

What could be the problem? Mail server infos are double checked and ok. Should I restart the pretix server?

### Expected behaviour

see above

### Steps to reproduce

see above

**Comment by raphaelm** [comment]
Have you checked the pretix.log log file on the server if it contains error messages, or the syslog/journal output of the worker processes?
